test/fungibletoken.test.js

```diff
@@ -0,0 +1,360 @@
+    await shallPass(
+      sendTransaction({
+        name: "mint_tokens",
+        args: [exampleTokenUserA, 100],
+        signers: [serviceAccount]
+      })
+    );
+    // Step 4: Transfer 50 tokens from account A to account B
+    await shallPass(
+      sendTransaction({
+        name: "transfer_tokens",
+        args: [50, exampleTokenUserB],
+        signers: [exampleTokenUserA]
+      })
+    );
+  })
+
+  // Fourth test burns tokens
+  test("should be able to burn tokens", async () => {
+    // Step 1: Mint tokens to the very example token admin account
+    await shallPass(
+      sendTransaction({
+        name: "mint_tokens",
+        args: [serviceAccount, 100],
+        signers: [serviceAccount]
+      })
+    );
+    // Step 2: Burn 50 tokens from the example token admin account
+    await shallPass(
+      sendTransaction({
+        name: "burn_tokens",
+        args: [50],
+        signers: [serviceAccount]
+      })
+    );
+  })
+});
+
+// Defining the test suite for the fungible token switchboard
+describe("fungibletokenswitchboard", ()=>{
+
+  // Variables for holding the account address
+  let serviceAccount;
+  let fungibleTokenSwitchboardUser;
+
+  // Before each test...
+  beforeEach(async () => {
+    // We do some scafolding...
+
+    // Getting the base path of the project
+    const basePath = path.resolve(__dirname, "../"); 
+		// You can specify different port to parallelize execution of describe blocks
+    const port = 8080; 
+		// Setting logging flag to true will pipe emulator output to console
+    const logging = false;
+
+    await init(basePath, { port });
+    await emulator.start(port, {logging});
+
+    // ...then we deploy the ft and example token contracts using the getAccountAddress function
+    // from the flow-js-testing library...
+
+    // Create a service account and deploy contracts to it
+    serviceAccount = await getAccountAddress("ServiceAccount");
+
+    await deployContract({ to: serviceAccount,    name: "FungibleToken"});
+    await deployContract({ to: serviceAccount,    name: "ExampleToken"});
+    await deployContract({ to: serviceAccount,    name: "FungibleTokenSwitchboard"});
+    
+    // Deployed at address which has the alias - fungibletokenswitchboardUser
+    fungibleTokenSwitchboardUser = await getAccountAddress("SwitchboardUser");
+  
+  });
+
+  // After each test we stop the emulator, so it could be restarted
+  afterEach(async () => {
+    return emulator.stop();
+  });
+
+  // First test checks if switchboard can be installed
+  test("should be able to setup switchboard", async () => {
+    // Only step: setup switchboard
+    await shallPass(
+      sendTransaction({
+        name: "switchboard/setup_account",
+        args: [],
+        signers: [fungibleTokenSwitchboardUser]
+      })
+    );
+  });
+
+  // Second test checks if switchboard user is able to add ft token vault capabilities
+  test("should be able to create and remove vault capability", async () => {
```

Comment:
I think need to change the test statement as it is not removing the capability it is only adding.